lib/ansible/modules/cloud/scaleway/scaleway_lb_facts.py

```diff
@@ -0,0 +1,129 @@
+short_description: Gather facts about the Scaleway load-balancer available.
+description:
+  - Gather facts about the Scaleway load-balancer available.
+version_added: "2.8"
+author:
+  - "Remy Leone (@sieben)"
+extends_documentation_fragment: scaleway
+options:
+
+  region:
+    description:
+    - Scaleway zone
+    required: true
+    choices:
+      - nl-ams
+      - fr-par
+'''
+
+EXAMPLES = r'''
+- name: Gather Scaleway organizations facts
+  scaleway_lb_facts:
+'''
+
+RETURN = r'''
+---
+scaleway_lb_facts:
+  description: Response from Scaleway API
+  returned: success
+  type: complex
+  contains:
+     "scaleway_lb_facts": [
+                {
+                    "backend_count": 0,
+                    "description": "Load-balancer used for testing scaleway_lb_facts ansible module",
+                    "frontend_count": 0,
+                    "id": "00000000-0000-0000-0000-000000000000",
+                    "instances": [
+                        {
+                            "id": "00000000-0000-0000-0000-000000000000",
+                            "ip_address": "10.14.229.145",
+                            "region": "fr-par",
+                            "status": "ready"
+                        },
+                        {
+                            "id": "00000000-0000-0000-0000-000000000000",
+                            "ip_address": "10.14.208.171",
+                            "region": "fr-par",
+                            "status": "ready"
+                        }
+                    ],
+                    "ip": [
+                        {
+                            "id": "00000000-0000-0000-0000-000000000000",
+                            "ip_address": "51.159.26.123",
+                            "lb_id": "00000000-0000-0000-0000-000000000000",
+                            "organization_id": "00000000-0000-0000-0000-000000000000",
+                            "region": "fr-par",
+                            "reverse": ""
+                        }
+                    ],
+                    "name": "lb_facts_ansible_test",
+                    "organization_id": "00000000-0000-0000-0000-000000000000",
+                    "region": "fr-par",
+                    "status": "ready",
+                    "tags": [
+                        "first_tag",
+                        "second_tag"
+                    ]
+                }
+            ]
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.scaleway import (
+    Scaleway,
+    ScalewayException,
+    scaleway_argument_spec,
+    SCALEWAY_REGIONS,
+    SCALEWAY_ENDPOINT)
+
+
+class ScalewayLbFacts(Scaleway):
+
+    def __init__(self, module):
+        super(ScalewayLbFacts, self).__init__(module)
+
+        region = module.params["region"]
+        self.module.params['api_url'] = "/".join([SCALEWAY_ENDPOINT,
+                                                  'lbaas/v1beta1/regions/%s' % region])
```

Comment:
The URL path should be customizable (or the API version), I guess `v1beta1` won't stay forever (meaning: the path will change without implying a new behavior).

Reply:
The problem is that region is also inside the path :/ How could we have both flexibility without introducing region templating from the user side?